Expecto/Expect.fs

```diff
@@ -304,8 +304,13 @@ let stringStarts subject prefix message =
       "%s. Expected subject string to start with the prefix. Differs at position %i with subject '%c' and prefix '%c'.%s"
       message i s p (printVerses " prefix" prefix "subject" subject)
 
-/// Expects the two values to equal each other.
-let equal (actual : 'a) (expected : 'a) message =
+let colorizeString (color:ConsoleColor) text : string =
+  sprintf "%s%s%s"
+      (ANSIOutputWriter.colourText (ANSIOutputWriter.getColour()) color)
+      text
+      ANSIOutputWriter.colourReset
+
+let equalDiffer differ (actual : 'a) (expected : 'a) message =
```

Comment:
Let's chat about configuration

Reply:
Also, format params like `(myParam: int)`, not `(myParam:int)` and not `(myParam : int)`

Reply:
I agree. This is just haste-work ;) I'll fix formatting

Reply:
Btw. I think the reason I formatted it this way was because the existing functions are formatted that way...?

Reply:
```
/// Expects the two values not to equal each other.
let notEqual (actual : 'a) (expected : 'a) message =
  if expected = actual then
    failtestf "%s. Actual value was equal to %A but had expected them to be non-equal."
        message actual
```

E.g. this one is formatted like this ^